common/v2/features/Dashboard/components/WalletBreakdown/BreakdownChart.tsx

```diff
@@ -113,6 +113,14 @@ export default class BreakdownChart extends Component<BreakdownChartProps> {
     this.props.setSelectedAssetIndex(index);
   };
 
+  public handleMouseLeave = (_: any) => {
+    if (this.isChartAnimating) {
+      return;
+    }
```

Comment:
To facilitate readability prefer defending on a single line.
`if (this.isChartAnimating) return;`